### PR TITLE
add tabby v10.0.5 with git auto-update config

### LINKED ISSUE #8009: [Request] Add tabby
**Library name:** tabby
**Git repository url:** https://github.com/cferdinandi/tabby

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t, please help me to review this pull request, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/cferdinandi/tabby
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 199
   - Count of watchers: 13
   - Count of forks: 23
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8009, cc @cferdinandi

### CODE REVIEW

**Comment by cferdinandi** [comment]
@nolsherry I am the author of this package, not @PeterDaveHello. Please do not take that as an endorsement of this PR.

**Comment by pvnr0082t** [comment]
@nolsherry Please fix the problem of previous PR first.

File: ajax/libs/tabby/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "tabby",
+  "filename": "tabby.min.js",
+  "version": "10.0.5",
+  "description": "Simple, mobile-first toggle tabs.",
+  "author": {
+    "name": "Chris Ferdinandi",
+    "url": "http://gomakethings.com"
+  },
+  " autoupdate": {
```
Comment (pvnr0082t) [inline ajax/libs/tabby/package.json]: There is an unnecessary space here.

**Comment by pvnr0082t** [comment]
@nolsherry You need to keep the structure as same as upstream. There are `js` and `css` folders under  `dist` folder.

**Comment by pvnr0082t** [comment]
@nolsherry And I wonder why the committer is @kennynaoh.

**Comment by nolsherry** [comment]
@pvnr0082t, I have fixed the structure error,and ```js```and ```css``` folders under ```dist``` folder.
Please help me to review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/tabby/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cferdinandi/tabby"
+  },
+  "keywords": [
+    "tabby",
+    "simple toggle tabs",
+    "toggle tabs style"
```
Comment (PeterDaveHello) [inline ajax/libs/tabby/package.json]: That's not what keywords does.

**Comment by nolsherry** [comment]
@pvnr0082t, I have revised the ```keywords```, please help me to review the pull request, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**